packages/react/src/components/OverflowMenu/OverflowMenu-test.js

```diff
@@ -80,6 +80,10 @@ describe('OverflowMenu', () => {
         0
       );
     });
+    it('should specify light version as expected', () => {
+      rootWrapper.setProps({ light: true });
+      expect(rootWrapper.props().light).toEqual(true);
```

Comment:
should we also check that the `.bx--overflow-menu--light` class is applied?

Reply:
Good point. I'll add it, thanks.